src/components/Navbar/Navbar.tsx

```diff
@@ -60,6 +61,22 @@ export const Navbar: React.FC<NavbarProps> = ({
   );
 
   const appWidth = useAppWidth();
+  const [maxHeight, setMaxHeight] = React.useState(0);
+  const containerRef = React.useRef<HTMLDivElement>(null);
+
+  useResizeObserver(containerRef, ({ contentRect }) => {
+    if (contentRect.height > 0 || fullscreenHandle.active) {
```

Comment:
To keep the height that was used before fullscreen, we can move the check for fullscreenHandle.active to where we set the maxHeight style inline.

```suggestion
    if (contentRect.height > 0) {
```